packages/test/package.json

```diff
@@ -27,6 +27,7 @@
     "@temporalio/proto": "file:../proto",
     "@temporalio/worker": "file:../worker",
     "@temporalio/workflow": "file:../workflow",
+    "@opentelemetry/exporter-collector-grpc": "^0.24.0",
```

Comment:
The jaeger-exporter dependency should be removed from https://github.com/temporalio/sdk-node/blob/bdbd63b75f341a8d0008b7c92686a7259bf11411/package.json#L10